src/test/resources/fixtures/namespace/request_no_description.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ownerName": "analytics",
+  "description": null
```

Comment:
This is a really good set of tests, particularly for user input very close to the edge. Glad we're doing these!

That said, itseems like this file is intended to be what the user sends over. If so, for me, when I'd like to create a request using cUrl or another REST client, and I don't have a description for something, I usually just omit instead of sending over null or even something blank. This is different than the case you're testing here.

I think this set of tests would definitely benefit from testing the case of the description field being omitted entirely (both in this particular example and in other situations w/o a description).

For example, in this case, I think a more representative test case would be to just send over the ownerName and verify the result is what you expect.

Reply:
I've removed the static json fixtures representing req/resp payloads in favor of using [`ApiModelGenerator`](https://github.com/MarquezProject/marquez/blob/feature/to-from-json-http-req-tests/src/test/java/marquez/api/models/ApiModelGenerator.java) to ensure valid values and inlining json within each test.

> I don't have a description for something, I usually just omit instead of sending over null or even something blank. This is different than the case you're testing here.

It would be difficult to differentiate between a field that was set as _null_ vs one that was omitted all together. We'd be testing how Jackson handles such a case and personally don't find much value in adding such tests (`Optional` fields already default to _null_)